courses/machine_learning/deepdive2/launching_into_ml/solutions/explore_data.ipynb

```diff
@@ -0,0 +1,1980 @@
+{
```

Comment:
<p>Same story about 4 spaces instead of 2 :)</p><p>Apply throughout.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/654/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Done</p>